### DISCUSSION TITLE
Mars 3 / ctb v4 file support?

### DISCUSSION THREAD

**Discussion by Mjspikes** [rudetrooper/Octoprint-Chituboard#52] [Q&A]
Good day, I have a mars 3 which i have installed the plugin on and followed the steps to get going.
I cant seem to even move the machine or print from the plugin.
I know i am using the v4 files AKA new firmware is there any new on the v4 support?

**Comment by johnspeakEPS** [comment]
No, the author says not:

> Plugin might not work if you've updated your Elegoo Mars printer to the newest firmware due to issues with Chitu3d encrypting their files so users are forced to use Chitubox 1.9.0. **I'm not planning on incorporating the non FOSS chitubox SDK into an AGPLv3 licensed plugin.**
